src/main/java/hr/inovatrend/petManager/Controller/UserController.java

```diff
@@ -54,4 +54,15 @@ private String userEdit(@PathVariable("id") Long id, @ModelAttribute User user)
     }
 
 
+    @GetMapping("/delete/{id}")
+    private String deleteUser(@PathVariable("id") Long id) {
+
+        User user = userService.getUser(id);
```

Comment:
Zašto se dohvaća user? Koliko vidim delete metoda briše po Idu, a ne po objektu usera.

Reply:
Popravljeno 
[053274d](https://github.com/Lora9299/Pet-Manager/pull/5/commits/053274db14f9e41aec28900ed09bc34ea28d7549)